### DISCUSSION TITLE
How to overlap plots?

### DISCUSSION THREAD

**Discussion by ubaldot** [hyOzd/serialplot#56] [Q&A] [answered]
Say that from an Arduino board I am sending something like this (comma separated values):
```
    int sensorValue = analogRead(A0);
    
    // Send over serial
    Serial.print(sensorValue);
    Serial.print(",");
    Serial.print(sensorValue/2);
    Serial.print(",");
    Serial.print(sensorValue/4);
    Serial.print("\n\r");
```
I can plot the data on 3 different graphs (one per channel). Fine.

But what if I want to plot sensorValue and sensorValue/2 on the first graph and sensorValue/4 on the second graph?
How to do that?

**Comment by hyOzd** [comment]
Unfortunately, this is not possible at the moment.

**Comment by ubaldot** [comment]
Ok, shall I open an issue about that (if it is planned to be done)? Or is there any workaround (from the screenshots I can see multiple plots on the same graph)?

**Reply by hyOzd** [reply, marked as answer]
Sure you can open a feature request.
